src/Interfaces/IGetQueueAvailabilityOptionalParams.ts

```diff
@@ -0,0 +1,16 @@
+import InitContext from "../Model/InitContext";
+
+export default interface IGetQueueAvailabilityOptionalParams {
+  /**
+   * JWT token from the portal
+   */
+  authenticatedUserToken?: string;
```

Comment:
Does this endpoint has an auth'ed endpoint? If not, then the `OptionalParams` is not needed.

Reply:
Yes, this will be an authenticated end point